test/test_jit.py

```diff
@@ -497,6 +498,13 @@ def runAndSaveRNG(self, func, inputs, kwargs=None):
         return results
 
 
+# has to be at top level or Pickle complains
+class FooToPickle(torch.nn.Module):
```

Comment:
why this class is not under the test case?

Reply:
```
# has to be at top level or Pickle complains
```
Limitation in pickle.